tss-esapi/tests/integration_tests/structures_tests/buffers_tests/nonce_tests.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Contributors to the Parsec project.
```

Comment:
```suggestion
// Copyright 2021 Contributors to the Parsec project.
```

Reply:
There are quite a few other files below that need the same. It's a bit problematic to tell which files were actually just copied, some are clearly marked as "renamed only" by Github, but I don't know which of these are renamed and slightly modified

Reply:
No, it's only for new files, but when files get broken up and shuffled around it gets difficult to figure out which ones are new and which ones were only moved about

Reply:
I thought I was really careful moved things around with git and created new ones when needed. So the ones with 2020 should only be files that have moved. And files with 2021 should be new files that I created. I will have a look at it again and see.

Reply:
I looked at this and it is a just a rename with a few modifications.
```
git diff main:tss-esapi/tests/structure_buffers_nonce_tests.rs HEAD:tss-esapi/tests/integration_tests/structures_tests/buffers_tests/nonce_tests.rs 
```

Reply:
So it should be 2020

Reply:
Sorry for the tedious ask and thanks for checking it out!